array.go

```diff
@@ -1,6 +1,7 @@
 package mruby
 
 // #include "gomruby.h"
+// mrb_int _mrb_ary_len(mrb_value ary) { return RARRAY_LEN(ary); }
```

Comment:
Please put this definition in `gomruby.h` instead of here.

Reply:
Yep, I thought about this one after I went to bed.